packages/report/index.js

```diff
@@ -183,22 +183,17 @@ function renderCSV ({ users, userUploads }, dest, options = {}) {
 }
 
 /**
- * Create a new Fauna DB client instance from the passed environment variables.
+ * Create a new DB client instance from the passed environment variables.
```

Comment:
This package assumes it can pass graphql queries to the DB client as it was written in a time of Fauna. There is a good chunk more work to do to make this package work with postgres, and it's not clear if we still need it now we have a /metrics endpoint for prometheus metric tracking.

@alanshaw should `report` be refactored to work with postgres or removed as no longer needed?

Reply:
This is a good point. The same is true of `add-mock-deals.js` and `get-auth-token.js`. Presumably both of those want to be kept (and therefore refactored)?

Reply:
Just remove this whole directory.